stdlib/Michelson.ju

```diff
@@ -0,0 +1,143 @@
+  -- datatypes
+  let option    = %Michelson.option
+  let list      = %Michelson.list
+  let set       = %Michelson.set
+  let operation = %Michelson.operation
+  let contract  = %Michelson.contract
+  let pair      = %Michelson.pair
+  let either    = %Michelson.either
+  let lambda    = %Michelson.lambda
+  let map       = %Michelson.map
+  let big-map   = %Michelson.big-map
+
+  -- primitive types
+  let key       = %Michelson.key
+  let unit      = %Michelson.unit
+  let signature = %Michelson.signature
+  let chain-id  = %Michelson.chain-id
+  let int       = %Michelson.int
+  let nat       = %Michelson.nat
+  let string    = %Michelson.string
+  let bytes     = %Michelson.bytes
+  let mutez     = %Michelson.mutez
+  let bool      = %Michelson.bool
+  let hash-key  = %Michelson.hash-key
+  let timestamp = %Michelson.timestamp
+  let address   = %Michelson.address
+
+  -- arithmetic functions
+  sig add : (x: type) -> x -> x -> x
+  let add = %Michelson.add
+
+  sig mul : (x: type) -> x -> x -> x
+  let mul = %Michelson.mul
+
+  sig sub : (x: type) -> x -> x -> x
+  let sub = %Michelson.sub
+
+  sig div : (x: type) -> x -> x -> x
+  let div = %Michelson.div
+
+  sig abs : (x: type) -> x -> x -> x
+  let abs = %Michelson.abs
+
+  -- boolean functions
+  sig and' : (x: type) -> x -> x -> x
+  let and' = %Michelson.and
+
+  sig or  : (x: type) -> x -> x -> x
+  let or = %Michelson.or
+
+  sig xor : (x: type) -> x -> x -> x
+  let xor = %Michelson.xor
+
+  sig not : (x: type) -> x -> x -> x
+  let not = %Michelson.not
+
+  -- other functions
+  sig () : unit
+  let () = %Michelson.unit
+
+  sig now : timestamp
+  let now = %Michelson.now
+
+  sig amount : (x : let) -> x -> ()
+  let amount = %Michelson.amount
+
+  sig balance : (x : let) -> address -> x
+  let balance = %Michelson.balance
+
+  sig sha256 : (x: type) -> x -> hash-key
+  let sha256 = %Michelson.sha256
+
+  sig sha512 :
+  let sha512 = %Michelson.sha512
+
+  sig blake2b :
+  let blake2b = %Michelson.
+
+  sig source : address
+  let source = %Michelson.source
+
+  sig sender : address
+  let sender = %Michelson.sender
+
+  sig implicit-account : address
+  let implicit-account = %Michelson.implicit-account
+
+  sig create-contract : () -> signature
+  let create-contract = %Michelson.create-contract
+
+  sig transfer-tokens : (x: let) -> address -> x -> hash-key
+  let transfer-token = %Michelson.transfer-tokens
+
+  sig set-delegate : address -> ()
+  let set-delegate = %Michelson.set-delegate
+
+  sig none : either () a
+  let none = %Michelson.none
+
+  sig some : b -> either a b
```

Comment:
Same here

Reply:
probably best to use b for the second argument of either (doesn't matter, just a nit!)